tracing-core/src/span.rs

```diff
@@ -195,6 +197,11 @@ impl<'a> Attributes<'a> {
     pub fn is_empty(&self) -> bool {
         self.values.is_empty()
     }
+
+    /// Returns the field set in this `Attributes`.
+    pub fn field_set(&self) -> &FieldSet {
+        self.values.field_set()
+    }
```

Comment:
There's a similar public accessor on the `Metadata` type, but it's just called `fields()` rather than `field_set()`:
https://github.com/tokio-rs/tracing/blob/1e09e50e8d15580b5929adbade9c782a6833e4a0/tracing-core/src/metadata.rs#L146-L149
Let's use the same naming here --- I know the `ValueSet` method is called `fields_set()`, but that's not part of the public API, while `Metadata::fields` is. I think the public method naming should be consistent.
```suggestion
    pub fn fields(&self) -> &FieldSet {
        self.values.field_set()
    }
```